### PR TITLE
Add AnimesOnlineGames Presence

### LINKED ISSUE #1904: Animes Online | animesonline.games
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [x] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** Koalin#6339 

**What's the name of the service?** www.animesonline.games | AnimesOnline.Games

**What should the Presence display?** Title of anime and time left
**If possible, please provide a logo for the service (512x512 minimum)**  https://cdn.discordapp.com/attachments/725820850363826327/736972600764268615/discord.png

### LINKED ISSUE #3534: Animes Online Games | animesonline.games
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [x] Is the site locked to a specific country/region? Japão
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 
O RENEGADO#5028

**What's the name of the service?** 
[animesonline.games](https://animesonline.games) | Animes Online Games

**What should the Presence display?** 
must show the anime you are watching at the moment, what is the episode and the elapsed time "If possible the time"
Just like other sites (e.g. Crunchyroll, Netflix)

**If possible, please provide a logo for the service (512x512 minimum)** 
animesonlinegames

### PR DESCRIPTION
**Pull request by vahya**
Resolves PreMiD/Presences#1904, resolves PreMiD/Presences#3534

### CODE REVIEW

**Comment by vahya** [comment]
Screenshot 2021-06-30 024735
Screenshot 2021-07-03 193435
Screenshot 2021-07-03 193502
Screenshot 2021-07-03 193531

File: websites/A/AnimesOnline.Games/presence.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    largeImageKey: "logo_anime",
+    startTimestamp: browsingStamp
+  },
+    { pathname } = document.location,
+    { search } = document.location,
+    fav: HTMLDivElement = document.querySelector(".favFixed");
+
+  if (fav && fav.style.display === "block")
+    presenceData.details = "Olhando os favoritos";
+  else {
```
Comment (ririxidev) [inline websites/A/AnimesOnline.Games/presence.ts:17]: plz don't use if else

File: websites/A/AnimesOnline.Games/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "service": "AnimesOnline.Games",
+  "author": {
+    "name": "वाह्य",
+    "id": "819696394180100106"
+  },
+  "category": "anime",
+  "logo": "https://i.imgur.com/P4t78Oz.png",
```
Comment (ririxidev) [inline websites/A/AnimesOnline.Games/dist/metadata.json:9]: it has to be square!

**Review by rxri — CHANGES_REQUESTED** [review]

File: websites/A/AnimesOnline.Games/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "service": "AnimesOnline.Games",
```
Comment (ririxidev) [inline websites/A/AnimesOnline.Games/dist/metadata.json:3]: If they changed url, I would name it now
`AnimesOnlineGames` without dot.

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/A/AnimesOnline.Games/dist/metadata.json (hunk 3 above)
Comment (vahya) [reply websites/A/AnimesOnline.Games/dist/metadata.json:3]: Yeah, that seems like the reasonable thing to do

**Review by rxri — APPROVED** [review]
guess that's fine then

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**